src/js/chat/helpers.js

```diff
@@ -977,6 +977,16 @@ exports.translate = function($element, sender, text) {
     });
 };
 
+exports.followDate = function(user, channel) {
+    bttv.TwitchAPI.get('users/' + user + '/follows/channels/' + channel).done(function(data) {
+        var m = moment(data.created_at);
+        reply = 'Followed ' + m.fromNow() + ' (' + m.format('LLL') + ')';
```

Comment:
missing `var` ?

also maybe make the response more verbose, include the channel name for clarity, maybe accept specifying a channel as a param, default to current channel